### ISSUE TITLE
bin.osx: line 1: syntax error near unexpected token `newline' ./SDVMMR.bin.osx: line 1: `<!DOCTYPE html>'

### ISSUE THREAD

**Issue by yuukiw** [OutOfOrder/MonoKickstart#5]
i used the precompiled version and renamed them but this error always appears on mac devicses.

**Comment by flibitijibibo** [comment]
Is... is that an html page? Can you open that bin.osx file with a web browser? It should be a Mac executable.

**Comment by yuukiw** [comment]
yeah it seems like firefox saved that one file as a html instead of the bin for some reasson
